verifact/forum/admin.py

```diff
@@ -1,7 +1,7 @@
 from django.contrib import admin
-from .models import Question
+from .models import Question, Answer
 
 
-@admin.register(Question)
+@admin.register(Question, Answer)
```

Comment:
I think you should create a separate class for the AnswerAdmin as its fields differ from Question.